device-types/Cisco/MDS-C9148S.yaml

```diff
@@ -0,0 +1,138 @@
+---
+manufacturer: Cisco
+model: MDS 9148S
+slug: ds-c9148s
+part_number: DS-C9148S
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight:
+  - value: 17.42
+    unit: lb
+comments: '[Cisco MDS 9148S 16G Multilayer Fabric Switch Data Sheet](https://www.cisco.com/c/en/us/products/collateral/storage-networking/mds-9148s-16g-multilayer-fabric-switch/datasheet-c78-731523.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+module-bays:
+  - name: Slot 1
+    label: DS-C9148S-K9-SUP
+    position: '1'
+    desciption: '2/4/8/16 Gbps FC/Supervisor'
+  - name: Slot 2
+    position: '2'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Power Supply'
+  - name: Slot 3
+    position: '3'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Power Supply'
+  - name: Slot 4
+    position: '4'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 5
+    position: '5'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 6
+    position: '6'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
+  - name: Slot 7
+    position: '7'
+    desciption: 'MDS 9148S 16G 48 FC (1 Slot) Chassis Fan Module'
+interfaces:
+  - name: mgmt0
+    type: 1000base-t
+    mgmt_only: true
+  - name: fc{module}/1
```

Comment:
Any reason these have `{module}` in them? This is a device definition file, not a module definition file. If these are not modular ports then they should have the actual value, i.e. `fc1/1`. Also, if you run a `show config` or other applicable command, this should read exactly as the devices config does. On a 9200L for instance, even though you can configure a port by using `int gi1/0/1` it should be typed in this definition file as `GigabitEthernet1/0/1`. I have never used one of these devices so I dont know, just want to mention that in case it applies here.